Classes/Units/Controllers/ItemList.js

```diff
@@ -6,25 +6,54 @@ var ItemList = {
 			
 		}
 	},
-	3340: class Item { // yellow trinket
-		id = 3340;
-		static GoldCost = 0;
-		static isTrinket = true;
-		use(target = undefined){
-			
-		}
+	1001: class Item { // Boots of Speed
+		id = 1001;
+		static GoldCost = 325;
+		static GoldSell = 227;
```

Comment:
As it is 70% of GoldCost we can remove it.
Just note to have it in mind: gold values are floatic, so GoldSell for `Boots of Speed` should be `227.5`.

Reply:
Need to round the result.
How can you manage player's golds if the value is a float? In anycase need to round it.

http://ddragon.leagueoflegends.com/cdn/4.20.2/data/en_US/item.json (data.id[1001])

Reply:
Gold is floatic anyway:
- https://github.com/Karmel0x/LeagueEmulatorJS/blob/a8fcc93dced1b64877db985e7cee75a8add773bf/Packets/S2C/0x22-ADD_GOLD.js#L8
- https://github.com/Karmel0x/LeagueEmulatorJS/blob/a8fcc93dced1b64877db985e7cee75a8add773bf/Packets/LOW_PRIORITY/0xC4-CHAR_STATS.js#L123
- passive gold gain per second is 1.9

But if it's on `ddragon`, it may be correct that they are using `int` for item gold sell amount.

Reply:
I did goldSell variable from ddragon datas, can be that it use int values only because game client show it like an int value.
Like I said, need to decide how to round it (I will shortly test with boots, im curious to check how game client work with it)